### ISSUE TITLE
Recommendations

### ISSUE THREAD

**Issue by LegitSoulja** [LegitSoulja/OgarConsole#6]
Leave OgarConsole recommendations below to be left with discussions.

**Comment by Andrews54757** [comment]
I have created an installation plugin for this here: https://github.com/AJS-development/OgarUL-Plugin-Library. It is called ogarconsole installer and installs ogarconsole for you

**Comment by LegitSoulja** [comment]
I tested it and it works. Um could you have the updates coming from my repo instead of the fork? There's constant updates. And it would be cool for the plugin to check for sync, and update if theres a new update.

If not, then re-fork or pull or push or whatever how github works and update the fork since i've made a few changes. There will be constant changes with OgarCosole, so maybe a check for update with the plugin each time it runs? The main package.json in the master control the versions of the OgarConsole. Just sync and listen for the version json value. Compare them 2. Update if needed if they arent the same. Right now its v1.0.2 and I dont know why OgarUnlimitedLatest package.json contained version 1.0.3

**Comment by LegitSoulja** [comment]
But after all nice idea. Now people can install using the command 

> 'plugin add https://raw.githubusercontent.com/AJS-development/OgarUL-Plugin-Library/master/OgarConsole-Installer/files.txt'

**Comment by Andrews54757** [comment]
no, i cant as that way people can do `sudo node index.js` not `sudo node cmd.js` like usual. or you can add another folder here that has the same things from my fork

**Comment by LegitSoulja** [comment]
@Andrews54757 Will it work if i create another branch? Containing what your fork has? I'm not fimilar with github yet.

**Comment by Andrews54757** [comment]
yes

**Comment by LegitSoulja** [comment]
@Andrews54757 Can you work with this? https://github.com/LegitSoulja/OgarUL-OgarConsole-Installer

I've updated the links in index.js, and also files.txt. Was I supposed to? But thats there and I hope thats way you would like it

**Comment by LegitSoulja** [comment]
Ill make a plugin config for the plugin too here shortly.

**Comment by Andrews54757** [comment]
no, I mean something else, that is the plugin. the plugin downloads files from somewhere else. it downlad it from here https://github.com/AJS-development/OgarConsoleInstaller

**Comment by LegitSoulja** [comment]
Well you should edit your links to the links above in the src/ folder. That where the files will be installed from, and plus I can keep it updated from there. Or how do you want me to do this because im confused on how this works with you

**Comment by Andrews54757** [comment]
ok, so this https://github.com/AJS-development/OgarConsoleInstaller is different because it allows you to use `index.js` to start instead of `cmd.js`. the plugin from the official plugin library is a installer program. it doesnt contain the files. but the other link does. so the installer downloads the files from https://github.com/AJS-development/OgarConsoleInstaller. If you want to be able to edit directly, you need to copy https://github.com/AJS-development/OgarConsoleInstaller.

**Comment by Andrews54757** [comment]
the installer is a plugin that I created to install anything that requires advanced setup. (e.g. it wont work as a plugin alone)

**Comment by Andrews54757** [comment]
all the installer plugin needs is a url to download files from

**Comment by Andrews54757** [comment]
that url is https://github.com/AJS-development/OgarConsoleInstaller

**Comment by LegitSoulja** [comment]
Hmm, but um I've already done that. Ive added the plugin & the files it will be downloaded from which is here https://github.com/LegitSoulja/OgarUL-OgarConsole-Installer/tree/master/src

**Comment by Andrews54757** [comment]
Ok, i updated the plugin. to edit what the plugin downloads, all you need to do is to edit the src file in that repo

**Comment by LegitSoulja** [comment]
the files in src in that repo. ok gotcha
